contacts.module

```diff
@@ -160,6 +161,30 @@ function template_preprocess_contacts_dash_summary(array &$variables) {
   }
 }
 
+/**
+ * Pre render callback for dashboard blocks to add edit link to title.
+ *
+ * @param $element
+ *   The block render array.
+ *
+ * @return array
+ *   The block render array.
+ */
+function contacts_dashboard_block_edit_link_pre_render($element) {
+  if (!empty($link = $element['#dashboard_label_edit_link'])) {
+    if ($element['content']['#title']) {
```

Comment:
Excellent, though the `empty($bar = call())` looks good to very odd to me... Wonder if some extra brevity there would make it read nicer!

Reply:
Not sure what you mean, something like this?

```
  if (!empty($element['#dashboard_label_edit_link'])) {
    $link = $element['#dashboard_label_edit_link'];
    if ($element['content']['#title']) {
```

Reply:
Yeah, I think so. Or we can use a null coalesce or adding default properties, so:
```php
if ($title = $element['#dashboard_label_edit_link'] ?: FALSE)
```
or
```php
$element += ['#dashboard_label_edit_link' => FALSE]
if ($element['#dashboard_label_edit_link'])
```